recipes/protobuf-c/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ProtobufcConan(ConanFile):
+    name = "protobuf-c"
+    license = "https://github.com/protobuf-c/protobuf-c/blob/master/LICENSE"
+    url = "https://github.com/protobuf-c/protobuf-c"
+    settings = "cppstd", "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+    description = "conan package for protobuf-c"
+    exports_sources = ["patches/**"]
+    _source_subfolder = "source_subfolder"
+    requires = [
+        "protobuf/3.19.2",
+    ]
+    options = {
+        "build_proto3": [True, False],
+        "build_protoc": [True, False],
+        "shared": [True, False],
+        "verbose_makefile": [True, False],
+    }
+    default_options = {
+        "build_proto3": True,
+        "build_protoc": True,
+        "shared": False,
+        "verbose_makefile": False,
+    }
+
+    def config_options(self):
+        if self.settings.compiler == 'gcc' and float(self.settings.compiler.version.value) >= 5.1:
+            if self.settings.compiler.cppstd:
+                tools.check_min_cppstd(self, 11)
```

Comment:
- this logic must go into `validate()`
- this is not how you can compare versions.
- why gcc >= 5.1 needs C++11 but not other compilers?